pkgs/tools/misc/fastfetch/default.nix

```diff
@@ -0,0 +1,81 @@
+{ chafa
+, cmake
+, dbus
+, dconf
+, fetchFromGitHub
+, glib
+, imagemagick_light
+, lib
+, libglvnd
+, libxcb
+, makeWrapper
+, ocl-icd
+, opencl-headers
+, pciutils
+, pkg-config
+, stdenv
+, vulkan-loader
+, wayland
+, xfce
+, xorg
```

Comment:
Xorg isn't needed for wayland-only users and fastfetch runs normally without it (I've tested). This should be optional.